sycl/test-e2e/Basic/work_group_size_prop.cpp

```diff
@@ -1,8 +1,5 @@
-// RUN: %{build} -o %t.out
+// RUN: %{build} --gcc-toolchain=/opt/slurm/gcc/13.2.0 -o %t.out
```

Comment:
`--gcc-toolchain=/opt/slurm/gcc/13.2.0` is this addition intended to be committed, or an artifact from testing? If so could you elaborate on why we need it.

Reply:
Whoops no that was local! Thanks for spotting that.

Reply:
Pushed an undo for that